### PR TITLE
Multiple updates

### PR DESCRIPTION
**Pull request by nkoeries**
Hello Arthur, 
MC: Update content and images.
PMC: Update content as per MC. 
NC: Update content. 
CHW-MBC: Update content and image.  
BD: Title change to Congenital Disorders.

### CODE REVIEW

**Comment by arthurattwell** [comment]
@nkoeries Thanks. This looks good -- but I won't be able to check every change, so I'll leave it to you and the team to make sure you're happy with the web and PDF outputs. 

To check web output:

1. in Terminal, in the `bettercare` folder, enter: `bundle exec jekyll s`
2. in your browser, go to [http://127.0.0.1:4000/bettercare/](http://127.0.0.1:4000/bettercare/)

During a quick check I did spot that you need to change the title of Birth Defects in `index.md` in the repo root folder, too ([here](https://github.com/nkoeries/bettercare/blob/gh-pages/index.md)). To double-check, you can do a 'Find in folder...' search in Sublime Text to look through the entire repo for instances of `Birth Defects` and `birth-defects`.

Once you've made that fix in your fork, can you confirm whether you're getting web and PDF output you want, before I merge?

File: index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ layout: index
...
 		
-		<a href="birth-defects/0-3-contents.html" title="Birth Defects"><img class="cover-thumb" src="birth-defects/images/cover-thumb.jpg" alt="Birth Defects" title="Birth Defects is for those who work with patients with birth defects. It covers modes of inheritance, medical genetic counselling, birth defects due to chromosomal abnormalities, single gene defects, teratogens and multifactorial inheritance."/></a>
+		<a href="congenital disorders/0-3-contents.html" title="Congenital Disorders"><img class="cover-thumb" src="congenital-disorders/images/cover-thumb.jpg" alt="Congenital Disorders" title="Congenital Disorders is for those who work with patients with congenital disorders. It covers modes of inheritance, medical genetic counselling, congenital disorders due to chromosomal abnormalities, single gene defects, teratogens and multifactorial inheritance."/></a>
```
Comment (arthurattwell) [inline index.md]: @nkoeries Typo here: you need a hyphen in `href="congenital-disorders"` for the link to work (i.e. click on cover > opens book). You can test the changes on your own fork by going to your fork's staging site at https://nkoeries.github.io/bettercare/

Can you fix that, test, and let me know if it's working?

Comment (nkoeries) [reply index.md]: @arthurattwell I have changed it and it isn't working. Will comb through it to see what I missed. Would it be possible to delay pushing through this commit? (see comment on initial pull request)

**Comment by nkoeries** [comment]
@arthurattwell would it be possible to only push through the commits of Maternal Care? 
The cover of congenital disorders still has to be amended and PMC is with Gerhard for checking.  David has asked that Newborn Care wait until he has come back (there are changes he wants to add).

**Comment by arthurattwell** [comment]
@nkoeries Ah, I see you had a stash/merge problem in git, which has broken your HTML. I sympathise! Git can get stuck during a merge or an update if it's trying to merge the same lines from different sources, and doesn't know whether to keep the incoming version or the existing version. 

Git leaves markers in your files to show where it got confused. The first marker is a series of `>`s that say 'this is the first version of these lines'. The second marker is a series of `=`s, which say 'now, here's the second version of these lines'. Then, at the end, a series of `<`s, which signal the end of the conflicted section. Beside the `>`s and `<`s is the name of the source of each version. 

When you fix this, you need to manually remove the markers Git left behind, and delete whichever version of the conflicted lines you don't want. Then commit the file.

Basically, in this case, it looks like you need to remove lines 15, 16, 17, 19 here: https://github.com/nkoeries/bettercare/blame/gh-pages/index.md

If you get stuck, let me know. I could probably fix it, but don't want to meddle with your files.

**Comment by nkoeries** [comment]
@arthurattwell Index: it is fixed and in working order.

**Comment by arthurattwell** [comment]
@nkoeries I've merged this in now even though there are still changes coming. It's fine for pending issues (like the cover for *Congenital Disorders*) to be on the staging site, as long as we don't move them to the live server till they're ready.
